src/index.html

```diff
@@ -1,13 +1,518 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>The Met</title>
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./images/favicon-32x32.png"
+    >
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="./images/favicon-16x16.png"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="body">
+    <div class="body__wrapper">
+      <header class="header">
+        <div class="container">
+          <div class="top-actions">
+            <a href="#" class="top-actions__logo-container">
+              <img
+                src="images/logo.png"
+                alt="The Met Logo"
+                class="top-actions__logo"
+              >
+            </a>
+            <div class="top-actions__icons">
+              <div
+                class="top-actions__icon
+                top-actions__icon--phone"
+              >
+                <div class="
+                  phones
+                  top-actions__phones
+                  items-phones"
+                >
+                  <div class="phones__item item-phones">
+                    <p class="phones__title item-phones__title">
+                      The Met Fifth Avenue
+                    </p>
+                    <a href="tel:+1 212-535-7710" class="phones__phone item-phones__phone">
+                      +1 212-535-7710
+                    </a>
+                  </div>
+
+                  <div class="phones__item item-phones">
+                    <p class="phones__title item-phones__title">
+                      The Met Cloisters
+                    </p>
+                    <a href="tel:+1 212-535-7710" class="phones__phone item-phones_phone">
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
                    <a href="tel:+12125357710" class="phones__phone item-phones_phone">
```